tests/unittests/benchmark/task/test_tasks.py

```diff
@@ -1,7 +1,19 @@
 #!/usr/bin/env python
 """Tests for :mod:`orion.benchmark.task`."""
+import inspect
 
-from orion.benchmark.task import Branin, CarromTable, EggHolder, RosenBrock
+import pytest
+
+from orion.algo.space import Space
+from orion.benchmark.task import Branin, CarromTable, EggHolder, HPOBench, RosenBrock
+
+try:
```

Comment:
Not really necessary, but you could use the class ImportOptional like [here](https://github.com/Epistimio/orion/blob/f01220c58864f81421bb9750067e8b8593f9b2d0/src/orion/algo/dehb/dehb.py#L23) and [here](https://github.com/Epistimio/orion/blob/develop/tests/unittests/algo/test_dehb.py#L13).

Reply:
And HPOBench will make require additional tests for singularity, and so would best be in a separate test module.

Reply:
Sure, let me see If I can change it to similar stuff as ImportOptional, I actually saw such ImportOptional code and was just trying to make it simple in the test cases as `profet`.

I have not figured out an easy way to install `singularity` in our test env, maybe I can try it a little bit.